Present.md

```diff
@@ -1,9 +1,11 @@
-In this section I will talk about where Faker is at currently in terms of its development. Its important to keep in consideration the past of Faker, in all its
-forms. 
+This section will focus on where Faker is currently developing. 
 
-Faker PHP was sunsetted due to design limitations. The creator of PHP faker, noted that the project was relatively large, sitting at 3 mb, and was used for 
-automated testing, which meant it was being downloaded quite frequently. To be specific, the creator estimated that the project was downloaded around 121 million
-times around the time of the sunset announcement. 121 M x 3 = 363 Million MB. 
+Let us start by looking at version 6 of Faker, which is demarked by the project being rewritten in typescript, adding documentation, setting up a CI/CD pipeline, and beginning to make major bug fixes regarding the implementation.
+Towards the end of version 6, new features such as random IMEI numbers and MongoDB ObjectId generation. 
```

Comment:
Towards the end of version 6, new features such as random IMEI numbers and MongoDB ObjectId generation were added.